quantum/bootmagic/bootmagic_lite.c

```diff
@@ -17,7 +17,7 @@
 
 /** \brief Reset eeprom
  *
- * ...just incase someone wants to only change the eeprom behaviour
```

Comment:
These changes should be pulled out and added to a seperate PR

Reply:
Removed these changes from this PR. I leave it to the core maintainers to correct at your convenience.